starter-code/js/index.js

```diff
@@ -1,23 +1,79 @@
-function deleteItem(e){
 
-}
+var shoppingCart = []
 
-function getPriceByProduct(itemNode){
 
+function Product(name, price) {
+  this.name = '';
+  this.quantity;
+  this.price;
 }
 
-function updatePriceByProduct(productPrice, index){
 
+function getTotalPrice(product) {
+  var productCount = document.querySelectorAll(".wrapper");
+  for (var i=0; i<productCount.length; i++) {
+    var price = Number(document.getElementsByClassName("price")[i].innerText);
+    var quantity = Number(document.querySelectorAll(".quantity")[i].value);
```

Comment:
Try to be consistent. If you use `querySelectorAll()` use for everything. If you use `getElementsByClassName()` use for everything.